source/_components/zha.markdown

```diff
@@ -116,3 +137,28 @@ To remove modemmanager from an Debian/Ubuntu host run this command:
 ```bash
 sudo apt-get purge modemmanager
 ```
+
+### Can't connect to usb device and using docker
+If you can't connect as you are using docker you must forward your device from the host machine to the docker instance. This can be achieved by adding the device mapping to the end of the startup string or ideally using docker compose:
```

Comment:
```suggestion
If you are using Docker and can't connect, you most likely need to forward your device from the host machine to the Docker instance. This can be achieved by adding the device mapping to the end of the startup string or ideally using docker compose.
```